app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -279,7 +279,7 @@ def self.raw_alerts_connect(hostname, port, username, password)
         :username => username,
         :password => password
       }
-      ::Hawkular::Alerts::AlertsClient.new(url, credentials)
+      ::Hawkular::Alerts::Client.new(url, credentials)
```

Comment:
If the code worked before, this change shouldn't break it. Nor the version bump to `2.9.0` @lucasponce it works because alerts don't enforce the hawkular tenant header, right?